node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -384,10 +418,10 @@ impl Display for ByteMagnitude {
     }
 }
 
-pub struct SQLParams<'a> {
-    table_unique_key_name: &'a str,
+pub struct SQLParams<'params> {
```

Comment:
In case we can get rid of the lifetime `params` here, it would make the whole code lot easier to read. I believe, it's safe to use the `String` in-place of `&str`. We just need to find a good way to get rid of it from the `params_except_wei_change`.

Reply:
I wrote the code with the goal to provide an easy-to-use interface from outside. This is the price, kind of. You're not supposed to come in contact with this code much. The second face of this is that I had some performance concerns, using references, hoping to make it more efficient. But that part is debatable. Anyway, when I was learning Rust extensively, I realised the community favour referencing and sometimes they say that using the owned objects - if not necessary - makes weak code. I don't know. 

If I see some string to be hardcoded I'm reluctant to convert it to String because I know it will never change and so the piece of data of our program, in memory, will be there whether I choose to upgrade it to dynamically approached object or if I just reference to that memory slot directly. Therefore I think it doesn't create a new space of strings hanging in the memory all the time. It has ever been there as part of our binaries and therefore no need to create it again.